src/pages/guides/endpoints/classifications/import-file.md

```diff
@@ -37,25 +37,25 @@ curl -X POST "https://analytics.adobe.io/api/{GLOBALCOMPANYID}/classification/jo
      -H "Content-Type: application/json" \
      -H "Authorization: Bearer {ACCESS_TOKEN}" \
      -d '{
-            "dataFormat": "tsv",
-            "encoding": "UTF8",
-            "jobName": "testsuite evar1 classifications",
-            "notifications": [
-                {
-                "method": "email",
-                "state": "completed",
-                "recipients": [
-                    john@example.com
-                ]
-                }
-            ],
-            "listDelimiter": ",",
-            "source": "Direct API Upload",
-            "keyOptions": {
-                "byte_length": 0,
-                "type": "string"
+          "dataFormat": "tsv",
+          "encoding": "UTF8",
+          "jobName": "testsuite evar1 classifications",
+          "notifications": [
+            {
+              "method": "email",
+              "state": "completed",
+              "recipients": [
+                "john@example.com"
+              ]
             }
-         }'
```

Comment:
Good catch. This is Tony's JSON but I am sure he will agree with it when he reviews.